content/zh/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,2124 @@
+reviewers:
+- erictune
+- deads2k
+- liggitt
```

Comment:
The above lines can be removed in localized versions because the mentioned people won't review this localized version.